docs/tutorial/part-one/index.md

```diff
@@ -359,13 +359,13 @@ npm install --global surge
 surge
 ```
 
-Next, build your site by running the following command in the terminal at the root of your site (tip: make sure you're running this command at the root of your site, in this case in the hello-world folder, which you can do by opening a new tab in the same window you used to run `gatsby develop`):
+Next, build your site by running the following command in the terminal at the root of your site (tip: make sure you're running this command at the root of your site, in this case in the hello-world folder, which you can do by opening a new tab in the same window you used to run `npm run develop`):
 
 ```shell
 gatsby build
```

Comment:
npm run build

Reply:
I think this one is correct. It's referring back to the window in which the `npm run develop` command was initially run.

Reply:
Why exactly? It should still be `npm run build` I think.

Reply:
> which you can do by opening a new tab in the same window you used to run `gatsby develop`):

Is the original phrasing from the docs. It's explaining where to run the build command as opposed to how to run the build command.

Reply:
But that original phrasing _should_ be changed. Otherwise this is confusing? I can see someone trying to type `gatsby build` which won't work.

You can tweak this, e.g. 

> running the command `gatsby develop` via the npm script `npm run develop`

Reply:
Totally agree it is confusing.

Reply:
I had missed command in the code block. Thought we were talking about line 362 :-/